test/data/pti/parser_test_k.raw

```diff
@@ -0,0 +1,42 @@
+0,   100.00, 33, 0, 0, 60.00     / PSS(R)E 33 RAW created by rawd33  SUN, JAN 17 2021  10:37
+TESTS OTHER NETWORK FETURES
+TESTS PARSER AND EXPORT OF ZONE, INTERAREA-TRANSFER AND OWNER DATA
```

Comment:
Are these comments for the motivations of this new case accurate?

Reply:
Maybe not. I Just use this case to see how to see the behavior of owners, zones and areas.
I did not use it in tests so it would be better to remove it.

Reply:
ok, let's remove it then.  I try to keep the number of test files to a minimum and only add new files when the existing library of cases can be show to be insufficient.